apps/bigcommerce/schedule/README.md

```diff
@@ -0,0 +1,449 @@
+# Schedule by Aligent - User Guide
+
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+2. [Core Features](#core-features)
+3. [Creating Scheduled Updates](#creating-scheduled-updates)
+4. [Managing Scheduled Updates](#managing-scheduled-updates)
+5. [Action Types Explained](#action-types-explained)
+6. [Advanced Features](#advanced-features)
+7. [Best Practices](#best-practices)
+8. [Troubleshooting](#troubleshooting)
+9. [FAQ](#frequently-asked-questions)
+10. [Glossary](#glossary)
+
+---
+
+## Getting Started
+
+### What is Schedule by Aligent?
+
+Schedule by Aligent is a powerful BigCommerce app that automates pricing changes, product visibility updates, and other e-commerce operations based on your store's timezone. It eliminates the need for manual updates at specific times, allowing you to plan and schedule changes in advance.
+
+### Installation
+
+1. **Install from BigCommerce App Marketplace**
+   - Navigate to the BigCommerce App Marketplace
+   - Search for "Schedule by Aligent"
+   - Click "Install" and follow the prompts
+   - The app will automatically configure required permissions
+
+2. **Initial Setup**
+   - Upon first launch, the app syncs your store data:
+     - Customer Groups
+     - Price Lists
+     - Products
+     - Categories
+   - This process may take a few minutes depending on your catalog size
+
+3. **Verify Timezone Settings**
+   - Check your store timezone: **Store Settings → Date & Time**
+   - All scheduled times use this timezone
+   - Ensure it's correct before creating schedules
+
+### Required Permissions
+
+The app requires these BigCommerce permissions:
+- **Products Read/Write**: For managing price lists and sale prices
+- **Customers Read**: For accessing customer groups
+- **Information and Settings Read**: For timezone configuration
+
+---
+
+## Core Features
```

Comment:
Do we plan to do some QA on the core features? Or just wait for customers to tell us if something doesn't work (or doesn't work as expected)?